### PR TITLE
Update SSR testbed

### PR DESCRIPTION
**Pull request by andy-hook**
I noticed our SSR testing app was a little outdated while using it to test some changes.

This PR simply updates some outdated primitive apis and adds the slider to the `ssr-testing` app.

### CODE REVIEW

File: ssr-testing/pages/index.tsx [hunk 1]
```diff
@@ -42,6 +42,9 @@ export default function HomePage() {
         <li>
           <Link href="/tooltip">Tooltip</Link>
         </li>
+        <li>
```
Comment (benoitgrelard) [inline ssr-testing/pages/index.tsx:45]: nit: I think those are alphabetically ordered.

**Review by benoitgrelard — COMMENTED** [review]
Nice catch on this! Just a tiny nit.

**Comment by benoitgrelard** [comment]
@andy-hook flaky tests again?

**Comment by andy-hook** [comment]
> @andy-hook flaky tests again?

Yep, kinda happy to see it fail as had a few others recently too. Been investigating and think I have a lead.

**Review by jjenzz — APPROVED** [review]

**Review by benoitgrelard — APPROVED** [review]

**Pull request merged**